Riot/Modules/TabBar/TabBarCoordinator.swift

```diff
@@ -285,7 +286,8 @@ final class TabBarCoordinator: NSObject, TabBarCoordinatorType {
             viewControllers.append(roomsViewController)
         }
         
-        if RiotSettings.shared.homeScreenShowCommunitiesTab && showCommunities {
+        let matrixSession = parameters.userSessionsService.mainUserSession?.matrixSession
+        if RiotSettings.shared.homeScreenShowCommunitiesTab && !(matrixSession?.groups().isEmpty ?? false) && showCommunities {
```

Comment:
Maybe we can create a `private var showCommunitiesTab: Bool { return ... }`

Reply:
I cannot because it's slightly different for each case but I created the following var we can reuse
```
    private var currentMatrixSession: MXSession? {
        return parameters.userSessionsService.mainUserSession?.matrixSession
    }
```

Reply:
Ok indeed there is a small difference. Maybe we can add small comments on tab removal lines